README.md

```diff
@@ -162,14 +162,17 @@ Worklet script (i.e. `reach.js`):
 ```js
 class SendReachReportOperation {
   async function run(data) {
-    // A toy model that only computes reach for users whose favorite color is red.
-    if (data["favorite-color"] != "red") {
+    // A toy model that only computes reach for users who haven't previously had a report sent.
+    if (await this.sharedStorage.get("report-sent") != "yes") {
       return;  // Don't send a report.
     }
 
     // The user agent will send the report to a default endpoint after a delay.
-    privateAggregation.sendCountDistinctReport({
-      bucket: (await this.sharedStorage.get("id"))});
+    privateAggregation.sendHistogramReport({
+      bucket: (await this.sharedStorage.get("id")),
+      key: data["campaign-id"]});
```

Comment:
`sendHistogramReport()` takes a bucket and value. I think we'd probably want the campaign ID as the bucket if we're measuring per-campaign reach (or some fixed bucket if we're measuring overall reach). The value would then be a constant (see [here](https://github.com/alexmturner/private-aggregation-api#:~:text=convertAgeToBucketId(demo%5B%22age%22%5D)%3B-,value%3A%20128,-%2C%20%20//%20A%20predetermined%20fixed) for an example). I don't think we need the per-browser ID any more as we're directly preventing duplicate reports.